crowd_anki/importer/import_dialog.py

```diff
@@ -0,0 +1,146 @@
+from collections import namedtuple
+from dataclasses import dataclass
+from enum import Enum
+
+from aqt.qt import *
+from typing import List, Tuple
+
+from .import_ui import Ui_Dialog as ConfigUI
+from ..config.config_settings import ConfigSettings
+
+from ..utils.utils import string_cs_to_list
+
+ConfigEntry = namedtuple("ConfigEntry", ["config_name", "default_value"])
+
+
+class ImportJsonSetting:
+    import_message: str
+    suggest_tag_imported_cards: bool
+    personal_fields: List[Tuple[str, str]] = []
+
+    class Properties(Enum):
+        IMPORT_MESSAGE = ConfigEntry("import_message", "")
+        SUGGEST_TAG_IMPORTED_CARDS = ConfigEntry("suggest_tag_imported_cards", False)
+
+    def __init__(self, settings_dict: dict):
+        for prop in self.Properties:
+            setattr(self, prop.value.config_name, settings_dict.get(prop.value.config_name, prop.value.default_value))
+
+        models = settings_dict.get("note_models", [])
+        for model_name in models:
+            personal_fields = models[model_name].get("personal_fields", [])
+            for pf in personal_fields:
+                self.personal_fields.append((model_name, pf))
+
+
+class ImportConfig:
+    personal_fields: List[Tuple[str, str]] = []
+    add_tag_to_cards: List[str] = []
+
+    use_header: bool = True
+    use_note_models: bool = True
+    use_notes: bool = True
+    use_media: bool = True
```

Comment:
without the dataclass annotation these would be the class variables not the object ones, which is probably what you intended